prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,49 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
+			nonMembers, emerr := CheckMembership(gc, ae.org, unassign)
+			if emerr != nil {
+				log.Printf("CheckMembership Error : %v", emerr)
+			} else if len(nonMembers) > 0 {
+				msg += fmt.Sprintf("\nFollowing github user(s) are not member of %v : %v", ae.org, strings.Join(nonMembers, ","))
+			} else {
+				msg += "\nOther Error. @kubernetes/test-infra-maintainers"
+			}
+
+			ic := github.IssueComment{
+				Body:    ae.body,
+				User:    github.User{Login: ae.login},
+				HTMLURL: ae.url,
+			}
+			if e2 := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponse(ic, msg)); e2 != nil {
+				return fmt.Errorf("Unassign err : %v, comment err: %v", msg, e2)
+			}
+			return errors.New(msg)
 		}
 	}
 	if len(assign) > 0 {
 		log.Printf("Adding assignees to %s/%s#%d: %v", ae.org, ae.repo, ae.number, assign)
 		if err := gc.AssignIssue(ae.org, ae.repo, ae.number, assign); err != nil {
-			return err
+			msg := fmt.Sprintf("Assign failed: %v.", err.Error())
+			nonMembers, emerr := CheckMembership(gc, ae.org, assign)
+			if emerr != nil {
+				log.Printf("CheckMembership Error : %v", emerr)
+			} else if len(nonMembers) > 0 {
+				msg += fmt.Sprintf("\nFollowing github user(s) are not member of %v : %v", ae.org, strings.Join(nonMembers, ","))
+			} else {
+				msg += "\nOther Error. @kubernetes/test-infra-maintainers"
```

Comment:
This should not be a part of the comment. `plugins.FormatResponse` will already add a link to the details block that lets people file issues in test-infra.